Intro Skip/SkipDaemon.cs

```diff
@@ -53,17 +53,22 @@ public void Initialize()
             float firstObjectTime = _initData.audioClip.length;
             float lastObjectTime = -1f;
 
+            int count = 0;
+
             foreach (var item in beatmapDataItems)
             {
                 if (item is NoteData note || (item is ObstacleData obstacle && IsObstacleInHeadArea(obstacle)))
                 {
+                    count++;
                     if (item.time < firstObjectTime)
                         firstObjectTime = item.time;
                     if (item.time > lastObjectTime)
                         lastObjectTime = item.time;
                 }
             }
 
+            if (count == 0) return;
```

Comment:
nit: Can you move the return statement to the next line? It improves readability.